lib/wit.js

```diff
@@ -38,11 +40,14 @@ function Wit(opts) {
     ;
   };
 
-  this.converse = (sessionId, message, context) => {
+  this.converse = (sessionId, message, context, reset) => {
     let qs = 'session_id=' + encodeURIComponent(sessionId);
     if (message) {
       qs += '&q=' + encodeURIComponent(message);
     }
+    if (reset) {
```

Comment:
not sure it's a good idea to have reset as a part of the converse function -- 

they seem different to me, mainly because the reset only needs the session id, nothing else (if I understood correctly).

In that case, we could do 

````
  this.reset(sessionId);
  this.converse(sessionId, message. context);
```

cc @blandinw @patapizza for thoughts